src/components/implementation/tmrmgr/simple/tmrmgr.c

```diff
@@ -29,11 +29,25 @@ struct tmr_info {
 };
 
 SS_STATIC_SLAB(timer, struct tmr_info, MAX_NUM_TMR);
-struct heap* timer_active;
```

Comment:
We only use typedefs and `*_t` names if they are value types. We use `struct blah` as the type when they cannot be passed by value reasonably. See the style guide. This is to make it easier to know how to pass around the type without having to look it up for correctness concerns.

If there was a reason to typedef it, maybe I'm wrong here, but this is not conventional.

Reply:
I read the related section in the style guide explains when not to most of the cases, thank you. Removed typedefs.